yourgov/components/Staff/lib/getDisplayData.ts

```diff
@@ -91,7 +90,10 @@ const sortData = (a: StaffMember, b: StaffMember, sort: GetDataSort) => {
 };
 
 /** Simulates the retrieval of data from an API */
-export async function getData(params: GetDataParams): Promise<GetDataResponse> {
+export async function getDisplayData(
+	staffMembers: StaffMember[],
+	params: GetDisplayDataParams
+): Promise<GetDataResponse> {
 	const { page, perPage } = params.pagination;
```

Comment:
Seems to be an issue. When you first load it says there are 95 items, but if you search for "ri", it only shows 6 results, but says 1 - 10 of 51 items 🤷‍♂️

Reply:
Hmm ok, thanks. Good spot. Paul mentioned a related bug when you delete all of the items on the last page, you get into a broken state. I'll look into this.

- [ ] Fix pagination WRT search + deletion